src/ansys/fluent/core/solver/flobject.py

```diff
@@ -1007,6 +1033,20 @@ class Command(Action):
 
     def __call__(self, **kwds):
         """Call a query with the specified keyword arguments."""
+        if hasattr(self, "file_name"):
+            file_purpose = self.file_name.get_attr(_InlineConstants.file_purpose)
+        elif hasattr(self, "file_name_list"):
+            file_purpose = self.file_name_list.get_attr(_InlineConstants.file_purpose)
+        elif hasattr(self, "filename"):
+            file_purpose = self.filename.get_attr(_InlineConstants.file_purpose)
+        print(f"\nfile_purpose = {file_purpose}\n")
+
+        if file_purpose == "input" and pypim.is_configured():
```

Comment:
`pypim.is_configured()` will not work in pyconsole. `remote_file_handler` can provide a method to check whether remote is configured or not.

Reply:
Yes, sure.

Reply:
@mkundu1 @seanpearsonuk 

I installed `ansys-platform-instancemanagement` in pyconsole as follow.

`D:\ANSYSDev\vNNN\commonfiles\CPython\3_10\winx64\Release\python\python.exe -m pip install ansys-platform-instancemanagement`

Now we can check `pypim.is_configured()` in pyconsole.

![image](https://github.com/ansys/pyfluent/assets/106588300/d2d304a5-a165-48e8-8146-3d1c6c08b87c)

We can request build team to install `ansys-platform-instancemanagement` in both `thirdparty` and `commonfiles` python as it is one of the basic requirement to support upload/download across all API.

Reply:
> `pypim.is_configured()` will not work in pyconsole. `remote_file_handler` can provide a method to check whether remote is configured or not.

@mkundu1 Is there any other reason due to which `pypim.is_configured()` will not work in pyconsole ?